dashboards/gemini-cli/gemini-cli-monitoring.json

```diff
@@ -83,33 +61,10 @@
             },
             "dataSets": [
               {
-                "breakdowns": [],
-                "dimensions": [],
-                "legendTemplate": "",
-                "measures": [],
-                "minAlignmentPeriod": "86400s",
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "outputFullDuration": false,
-                  "timeSeriesFilter": {
-                    "aggregation": {
-                      "alignmentPeriod": "86400s",
-                      "crossSeriesReducer": "REDUCE_COUNT",
-                      "groupByFields": [
-                        "metric.label.\"user_email\""
-                      ],
-                      "perSeriesAligner": "ALIGN_RATE"
-                    },
-                    "filter": "metric.type=\"custom.googleapis.com/gemini_cli/gemini_cli.session.count\" metric.label.\"user_email\"=monitoring.regex.full_match(\"^[a-zA-Z0-9][a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]*@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$\")",
-                    "secondaryAggregation": {
-                      "alignmentPeriod": "2592000s",
-                      "crossSeriesReducer": "REDUCE_COUNT",
-                      "groupByFields": [],
-                      "perSeriesAligner": "ALIGN_MEAN"
-                    }
-                  },
-                  "unitOverride": ""
+                  "prometheusQuery": "label_replace(count(sum by (user_email) (increase({\"__name__\"=\"custom.googleapis.com/gemini_cli/gemini_cli.session.count\", \"monitored_resource\"=\"generic_node\"}[30d])) > 0), \"metric_label\", \"Code Assist Monthly Active Users\", \"\", \"\")"
```

Comment:
it looks like you dropped a user_email regex filter from the filter version - just verifying that's intentional/ok?